tutorials/io/70_reading_eyetracking_data.py

```diff
@@ -0,0 +1,163 @@
+from mne.datasets import misc
+
+# %%
+fpath = misc.data_path() / 'eyetracking' / 'eyelink'
+raw = read_raw_eyelink(fpath / 'px_textpage_ws.asc',
+                       create_annotations=['blinks'])
+custom_scalings = dict(eyegaze=1e3)
+raw.pick_types(eyetrack=True).plot(scalings=custom_scalings)
+
+
+# %%
+# .. important:: The (0, 0) pixel coordinates are at the top-left of the
+#               trackable area of the screen. Gaze towards lower areas of the
+#               the screen will yield a relatively higher y-coordinate.
+#
+# Note that we passed a custom `dict` to the ``'scalings'`` argument of
+# `mne.io.Raw.plot`. This is because MNE's default plot scalings for eye
+# position data are calibrated for HREF data, which are stored in radians
+# (read below).
+
+
+# %%
+# Head-Referenced Eye Angle (HREF)
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+#
+# HREF position data measures eye rotation angles relative to the head. It does
+# not take into account changes in subject head position and angle, or distance
+# from the stimulus screen. This datatype might be preferable for analyses
+# that are interested in eye movement velocities and amplitudes, or for
+# simultaneous and EEG/MEG eyetracking recordings where eye position data are
+# used to identify EOG artifacts.
+#
+# HREF coordinates are stored in the ASCII file as integer values, with 260 or
+# more units per visual degree, however MNE will convert and store these
+# coordinates in radians. The (0, 0) point of HREF data is arbitrary, as the
+# relationship between the screen position and the coordinates changes as the
+# subject's head moves.
+#
+# Below is the same text reading recording that we plotted above, except a new
+# ASCII file was generated, this time using HREF eye position data.
+
+
+# %%
+fpath = misc.data_path() / 'eyetracking' / 'eyelink'
+raw = read_raw_eyelink(fpath / 'HREF_textpage_ws.asc',
+                       create_annotations=['blinks'])
+raw.pick_types(eyetrack=True).plot()
+
+# %%
+# Pupil Position
+# ^^^^^^^^^^^^^^
+#
+# Pupil position data contains (x, y) coordinate pairs from the eye camera.
+# It has not been converted to pixels (gaze) or eye angles (HREF). Most use
+# cases do not require this data type, and caution should be taken when
+# analyzing raw pupil positions. Note that when plotting data from a
+# ``Raw`` object containing raw pupil position data, the plot scalings
+# will likely be incorrect. You can pass custom scalings into the ``scalings``
+# parameter of `mne.io.Raw.plot` so that the signals are legible when plotting.
+
+# %%
+# .. warning:: If a calibration was not performed prior to data collection, the
+#              EyeLink system cannot convert raw pupil position data to pixels
+#              (gaze) or eye angle (HREF).
+
+# %%
+# Pupil Size Data
+# ---------------
+# Pupil size is measured by the EyeLink system at up to 500 samples per second.
+# It may be reported as pupil *area*, or pupil *diameter*. The pupil size data
```

Comment:
- does the data type (area or diameter) also depend on your EDF2ASC settings alone  or is there anything to choose in MNE? 
lets make that a bit clearer..

- maybe add "(i.e. the diameter of a circle/ellipse model fit to the pupil area)"  after "diameter" (or some other indication that there is an additional modelling step involved)

Reply:
> * does the data type (area or diameter) also depend on your EDF2ASC settings alone  or is there anything to choose in MNE?

the data type to report is only specified in EDF2ASC (mne will just detect which one was output). I can try to clarify that

> * maybe add "(i.e. the diameter of a circle/ellipse model fit to the pupil area)"  after "diameter" (or some other indication that there is an additional modelling step involved)

good idea, will do!

Reply:
see [9157d98](https://github.com/mne-tools/mne-python/pull/11152/commits/9157d98b59c9ee403ed6995c05e42a3fa55ab773)